manifests/init.pp

```diff
@@ -18,6 +18,7 @@
   Array[String]                           $modules,
   Array[String]                           $modules_base,
   Array[String]                           $modules_disabled,
+  Array[String]                           $post_component_modules,
```

Comment:
This break CI since no default value is provided.

If it makes sense to have different default values, please set them in hiera, otherwise I would suggest moving this line bellow (with the ones with default values) and set it as amn empty array.

While here you can also forbid empty strings which I think do not make sense for a module name:

```suggestion
  Array[String[1]]                        $post_component_modules = [],
```